packages/flame_lottie/pubspec.yaml

```
@@ -0,0 +1,24 @@
+name: flame_lottie
+description: Flame wrapper for Lottie by AirBnB. This package implements a bridge between Lottie and Flame, allowing to load and display Lottie animations.
+version: 0.0.1
+homepage: https://github.com/flame-engine/flame/tree/main/packages/flame_lottie
+funding:
+  - https://patreon.com/bluefireoss
+  - https://www.buymeacoffee.com/bluefire
+
+environment:
+  sdk: '>=2.18.1 <3.0.0'
```

```suggestion
  sdk: '>=2.17.0 <3.0.0'
```
Or does it have anything that required it to be 2.18?